src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -300,7 +300,7 @@ public Response deleteRoleFromUser(@Auth AuthUser authUser, @PathParam("userId")
       } else if (activeUser.hasUserRole(UserRoles.SIGNINGOFFICIAL)) {
         if (!UserRoles.isValidSoAdjustableRoleId(targetRole)) {
           throw new ForbiddenException(
-              "A Signing Official may only remove the following role ids: [6, 7, 8] ");
```

Comment:
Interesting that we don't have a role id of 6 any more ... that might have been the defunct `DATA_OWNER` role.